work/flow.rst

```diff
@@ -230,7 +230,7 @@ When an external user has created a pull request (which should be automatically
         # Assume PR #456 is the community member's PR with newly-created Jira ticket DM-98765
         git fetch origin pull/456/head:tickets/DM-98765
 
-    4. Build and test the ticket (locally and with :ref:`Jenkins CI system <workflow-testing>`) as normal, so that it can be tracked using our existing tooling. As part of this, push to your own ticket branch:
+    4. Checkout the ticket branch and build and test the ticket (locally and with :ref:`Jenkins CI system <workflow-testing>`) as normal, so that it can be tracked using our existing tooling. As part of this, push to your own ticket branch:
```

Comment:
Maybe this should include the`git checkout` command since we're being explicit about everything else?

Reply:
The existing git checkout is now step #3.